### ISSUE TITLE
combination of createOnBlur and load makes it impossible to select loaded option

### ISSUE THREAD

**Issue by jrochkind** [orchidjs/tom-select#72]
**Description**

With a tom-select on a text input, with maxItems 1, create/createOnBlur, and `load`ed options, if you try to select one of the options, it does a createOnBlur instead. 

**To reproduce**

```
<input id="textInput" type="text">

new TomSelect(input_el, {
      create: true,
      createOnBlur: true,
      maxItems: 1,
      load: function(query, callback) {
        callback([
          {value: "1", text: "value1"},
          {value: "2", text: "value2" }
        ]);
      }
    });
```

1. Go to https://jsfiddle.net/vs9bmkxa/

2. Click in the text box. 
3. Type "v" to get the value list to show up. 
4. Select either of the options that show up, `value1` or `value2`. Select by arrow keys and return OR by mouse clicking. 
5. See that nothing changes in the box, it still says `v` instead of `value1` or `value2`. If you open the drop-down again, you'll see that `v` now appears as an option -- the "create" option has been triggered, even though I didn't choose the "add" option, and DID mean to choose one of the other options. 

**Expected behavior**

After entering "v" and selecting one of the options, I expect:
* that option to be displayed in the visible box
* "v" NOT to have been added to the option list with "create"

**Additional context**
MacOS Chrome

**Comment by jrochkind** [comment]
Hm, I just noticed tom-select 1.5.0  was out. I don't seem to be able to reproduce on 1.5.0, I guess it has been fixed? Updated my fiddle to 1.5.0, and it no longer reproduces. It had been 1.4.3, I hadn't noticed 1.5.0 was out. 

will leave this open a bit longer while I confirm, but looks like you fixed it.

**Comment by oyejorge** [comment]
Yep, looks like it was fixed with 1.5
